istio.deps

```diff
@@ -4,7 +4,7 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "491c2e174bc6022e6af8beec7ee678cf48bbd130"
+    "lastStableSHA": "ae3bbc4313b45af63777a2588388796d74221cfd"
```

Comment:
I will do it in a follow-up, because I want to unblock our e2e tests, and I want to avoid additional commits, because the CI is very unstable recently.